textalyze.js

```diff
@@ -14,10 +6,17 @@
 function itemCounts(array) {
   let counts = new Map();
 
-  // Your code here.
-  // Run 'npm test' to see what tests need to pass.
+  for (const a of array) {
+    counts.has(a) ? counts.set(a, counts.get(a) + 1) : counts.set(a, 1);
+  }
 
   return counts;
 }
 
+const elements = ["a", "a", "a", "b", "b", "c"];
+console.log(`Count for [${elements}] is...`);
+itemCounts(elements).forEach((count, elem) => {
```

Comment:
Try to be consistent — you use `forEach` here but `for ... in ...` above.  The AirBnb style guide will tell you to use `forEach`.